php/docker/templates/nginx-conf.d/10-lua-init.conf

```
@@ -12,3 +13,20 @@ init_by_lua_block {
     google_credentials = gcs.init()
 }
 
+init_worker_by_lua_block {
+    prometheus = require("prometheus").init("prometheus_metrics")
+    metric_requests = prometheus:counter(
+        "nginx_http_requests_total", "Number of HTTP requests", {"host", "status", "cache_status"}
+    )
+    metric_request_time = prometheus:histogram(
+        "nginx_http_request_duration_seconds", "HTTP request request duration", {"host"}
+    )
+    metric_connections = prometheus:gauge(
+        "nginx_http_connections", "Number of HTTP connections", {"state"}
```

Let's rename `host` to `pod` or `server_name`. And each metric should have `pod` and `site_name` lable.
```suggestion
        "nginx_http_connections", "Number of HTTP connections", {"pod", "state"}
```